README.md

```diff
@@ -78,6 +78,7 @@ The following tables lists the configurable parameters of the Orchestrate chart
 | `global.labels`                    | Labels to add to all deploye                                                                             | `{}`                                                        |
 | `global.environment`               | Common environment variables for the API, Key Manager, Tx sender, Tx listener                            | `{}`                                                        |
 | `global.environmentSecrets`        | Common environment variables (as Kubernetes secrets)  for the API, Key Manager, Tx sender, Tx listener   | `{}`                                                        |
+| `global.existingSecretName`        | If specified, extra environment variables will be added from externally created secret                   | `nil`                                                       |
```

Comment:
To align with bitnami helm chart, could you name it `existingSecret` instead?

https://github.com/bitnami/charts/tree/master/bitnami/redis

Reply:
sure, will do

Reply:
done